recipe/meta.yaml

```diff
@@ -18,6 +21,7 @@ requirements:
   build:
     - {{ compiler('c') }}
     - {{ compiler('cxx') }}
+    - patch  # [py==311]
```

Comment:
If we build it for `win64` then `m2-patch  # [py==311]` should be added too

Reply:
addressed, thank you